server/utils/response.go

```diff
@@ -0,0 +1,51 @@
+package utils
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"time"
+)
+
+type errorResp struct {
+	success   bool
+	message   string
+	timeStamp time.Time
+}
+
+type successResp struct {
+	message string
+	success bool
+	data    interface{}
+}
+
+// Reusable err response function
+func ErrorResponse(w http.ResponseWriter, message string, status int) {
```

Comment:
As this func performs actions, it should start with a verb instead of a noun.

Same for line 34.